src/test/java/com/puppycrawl/tools/checkstyle/api/FullIdentTest.java

```diff
@@ -47,6 +47,13 @@ public void testToString() {
         Assert.assertEquals("Invalid full indent", "MyTest[15x14]", indent.toString());
     }
 
+    @Test
+    public void testGetDetailAst() {
+        final DetailAST ast = new DetailAST();
+        final FullIdent indent = FullIdent.createFullIdent(ast);
```

Comment:
This doesn't seem like a very detailed test unless we have one that is more complex than this. Please verify.
Can we have a test with more then 1 AST to show we stick with only the first one even if there is more.

Reply:
done